src/settings.ts

```diff
@@ -30,9 +31,9 @@ switch (env) {
   case 'staging':
     config = configStaging;
     break;
-  // case 'production':
```

Comment:
canyou make it that if `case: 'production'` it throws an error? This is what we do elsewhere.